### ISSUE TITLE
Better identification of Question title

### ISSUE THREAD

**Issue by tobydragon** [tobydragon/wizard#8]
Currently it strips all non-letters, and creates bad question titles (especially with activecode). Should be able to match from this field to the data from files

**Comment by cwormsl2** [comment]
It looked as though the colon ":" and the period "." were the only symbols in the question titles that didn't belong, so I removed those. However, while doing this I noticed a few things:
- In the xml from the site the type is mChoice and parsons for the respective problems but in Book.java it is either mchoicemf or parsonsprob
- A few questions are labeled as "actex" in Book.java but this type does not exist in the xml - the xml just uses "activecode" instead
- Some entries in Book.java are not actually questions. The entries labeled as activecode followed by an English sentence appear just as text in the online book, they are usually instructions preceding a question, but the question has a separate entry in Book.java. They also don't appear in the xml. 
- There are also entries in the xml that are not questions and as a result don't appear in Book.java such as every time a page is loaded, when a video is watched etc.
